Subtester/ProjectSettings/ProjectSettings.asset

```diff
@@ -145,11 +145,13 @@ PlayerSettings:
     enable360StereoCapture: 0
   isWsaHolographicRemotingEnabled: 0
   enableFrameTimingStats: 0
+  enableOpenGLProfilerGPURecorders: 1
   useHDRDisplay: 0
   D3DHDRBitDepth: 0
   m_ColorGamuts: 00000000
   targetPixelDensity: 30
   resolutionScalingMode: 0
+  resetResolutionOnWindowResize: 0
```

Comment:
Hmm so I'm not sure I understand it but should this be enabled? Seems like something we might want in case someone resizes it... Not sure though

Reply:
I've no idea tbh, this was just an automated change by Unity